components/add_groups_to_channel_modal/add_groups_to_channel_modal.tsx

```diff
@@ -114,9 +132,9 @@ export default class AddGroupsToChannelModal extends React.PureComponent {
         });
     }
 
-    handleSubmit = async (e) => {
+    handleSubmit = async (e: Event | GroupValue[] | undefined) => {
```

Comment:
```suggestion
    handleSubmit = async () => {
```

Reply:
@Willyfrog, thoughts regarding the comment?  @Ampit, many of us were out for the holidays, but will be catching up on PRs soon :)

Reply:
oh! sorry, seems I did comment but was caught as if it was a pending review or something.

your current approach seems to be the right one! sorry for the delay in the response